web/ugahacks-7/next.config.js

```diff
@@ -1,5 +1,11 @@
 module.exports = {
   reactStrictMode: true,
+  env: {
```

Comment:
You've exposed secrets to the internet.
These need to be regenerated and put into a file that is **not checked into source control.**